packages/elements-react/src/components/form/form.tsx

```diff
@@ -174,6 +185,26 @@ export type OryFlowComponents = {
   Page: {
     Header: ComponentType<OryPageHeaderProps>
   }
+  Settings: {
+    /**
+     * The SettingsSection component is rendered around each section of the settings.
+     */
+    Section: ComponentType<OryFormSectionProps>
+    /**
+     * The SettingsSectionContent component is rendered around the content of each section of the settings.
+     */
+    SectionContent: ComponentType<OryFormSectionContentProps>
+    /**
+     * The SettingsSectionFooter component is rendered around the footer of each section of the settings.
+     */
+    SectionFooter: ComponentType<OryFormSectionContentProps>
+
+    Oidc: ComponentType<OrySettingsOidcProps>
+    Webauthn: ComponentType<OrySettingsWebauthnProps>
+    Passkey: ComponentType<OrySettingsPasskeyProps>
+    Totp: ComponentType<OrySettingsTotpProps>
+    RecoveryCodes: ComponentType<OrySettingsRecoveryCodesProps>
+  }
```

Comment:
```
Screens: {
  Settings: {
    Section: ...
    Methods: {
      Oidc
      ...
    }
  }
}
```

Reply:
I am not really convinced by either. From our discussions, we still have the option of changing the <Settings /> components only, and I think that's probably the best solution. But non of these are great, IMO.

Reply:
https://github.com/ory/elements/pull/250